scripts/install-helm.sh

```diff
@@ -7,6 +7,8 @@ TEMP="/tmp/cephcsi-helm-test"
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 # shellcheck source=build.env
 [ ! -e "${SCRIPT_DIR}"/../build.env ] || source "${SCRIPT_DIR}"/../build.env
+# shellcheck disable=SC1091
+[ ! -e "${SCRIPT_DIR}"/rook.sh ] || source "${SCRIPT_DIR}"/rook.sh >/dev/null 2>&1
```

Comment:
I do not think you can safely `source` this, the file contains other things than just functions or other helpers. You can move the helper functions into their own file (`scripts/utils.sh` or something), and source that in both scripts.